ts/parts/Series.ts

```diff
@@ -6565,15 +6567,25 @@ H.Series = seriesType<Highcharts.LineSeries>(
             parent?: Highcharts.SVGElement
         ): Highcharts.SVGElement {
             var group = (this as any)[prop],
-                isNew = !group;
+                isNew = !group,
+                newAttrs: Highcharts.SVGAttributes;
 
             // Generate it on first call
             if (isNew) {
+                newAttrs = {
+                    zIndex: zIndex || 0.1 // IE8 and pointer logic use this
+                };
+
+                if (
+                    !this.chart.styledMode &&
+                    this.opacity &&
+                    this.opacity !== 1
+                ) {
+                    newAttrs.opacity = this.opacity;
+                }
```

Comment:
While reading directly from the options, this seems to do the job:
```js
if (!this.chart.styledMode) {
                    newAttrs.opacity = this.options.opacity;
                }
```

Also, the proposed logic doesn't allow users setting the opacity to 0.

Reply:
That said, it seems that neither of the solutions work with `Series.update`?

Reply:
`this.options.opacity;` was implemented in my first commit, then visual diffs show me that `series.opacity` option for venn and treemap is already used: https://api.highcharts.com/highcharts/series.treemap.opacity

So I decided to use `series.opacity` that can be removed in these series in `series.init`.

But since you are right, `series.update()` doesn't work, I will look for another solution (I guess plot groups are not removed on series.update).